### PR TITLE
loading spinner directive

### PR DESCRIPTION
**Pull request by jgabrielgv**

### CODE REVIEW

File: projects/components/src/lib/button.directive.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        role: 'button',
+        '[attr.tabindex]': '0',
+        '[attr.aria-disabled]': 'busy',
+    },
+})
+export class ButtonDirective implements OnChanges {
+    @Input()
+    variant? = 'secondary';
+    @Input()
+    busy?: boolean;
```
Comment (ruifang) [inline projects/components/src/lib/button.directive.ts]: This can just default to false, we'd treat null as false anyway.

**Review by alvartabe — APPROVED** [review]

File: projects/components/src/lib/button.directive.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    private loadingElement?: Node;
+    private svgContent = `
```
Comment (alvartabe) [inline projects/components/src/lib/button.directive.ts:18]: what happens if you just add here `<mz-spinner>` instead of `<svg>`?

Comment (jgabrielgv) [reply projects/components/src/lib/button.directive.ts:18]: As per request from Rui, we can't add this depedency

File: projects/components/src/lib/button.directive.ts (hunk 1 above)
Comment (jgabrielgv) [reply projects/components/src/lib/button.directive.ts]: Done

**Pull request merged**